package.json

```diff
@@ -1,36 +1,36 @@
 {
-  "name": "near-lake-framework",
-  "version": "1.0.6",
-  "description": "JS Library to connect to the NEAR Lake S3 and stream the data",
+  "name": "@near-lake-framework/monorepo",
```

Comment:
Also let's mark the root package as [private](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private) so that it is not accidentally published.